src/vsphere_cpi/spec/unit/cloud/vsphere/resources/storage_pod_spec.rb

```diff
@@ -20,6 +20,38 @@
     end
   end
 
+  describe '#inspect' do
```

Comment:
We don't need to test `#inspect`. This method should only be used in debugging code. If we're using it in production we're doing it wrong.

Reply:
Fixed